### PR TITLE
Add wheelchair_boarding to arrivals_departures

### PR DESCRIPTION
**Pull request by magnusburton**
Thought I’d add some accessibility data to arrivals_departures. This should conform to the Gtfs specification. I’m a big supporter for accessible public transport :) hope this helps!

### CODE REVIEW

File: lib/stop_times.js [hunk 1]
```diff
@@ -205,7 +205,11 @@ WITH stop_times_based AS NOT MATERIALIZED (
 			+ departure_time
 		) t_departure,
 		s.stop_id, stops.stop_name,
-		stations.stop_id station_id, stations.stop_name station_name
+		stations.stop_id station_id, stations.stop_name station_name,
+		CASE WHEN stops.wheelchair_boarding IS NULL OR stops.wheelchair_boarding =
```
Comment (derhuerst) [inline lib/stop_times.js:209]: Not sure if you find `coalesce()` more understandable, but we could write whole the expression as follows. What do you think?

```sql
		CASE WHEN
			stops.wheelchair_boarding IS NULL
			OR stops.wheelchair_boarding = 'no_info_or_inherit'
		THEN
			coalesce(stations.wheelchair_boarding, 'no_info_or_inherit')
		ELSE
			stops.wheelchair_boarding
		END AS wheelchair_boarding
```

Comment (magnusburton) [reply lib/stop_times.js:209]: Thanks! Coalesce would make this more readable

Comment (derhuerst) [reply lib/stop_times.js:209]: Would you also mind indenting your `CASE` expressions like (or similarly, I don't care about the exact indentation) I did it?

In `gtfs-via-postgres`, I keep the SQL statements this way to help with readability, and diff-ability.

**Review by derhuerst — CHANGES_REQUESTED** [review]
The `wheelchair_boarding` column is missing in the (root) `FROM` clause of the `arrivals_departures` view.

Also, for bonus points, could you add `wheelchair_boarding` to the `connections` view? 🙏

**Comment by magnusburton** [comment]
I'm going for the bonus points. Happy to help!

**Comment by derhuerst** [comment]
With one tiny caveat, this looks good. Thanks for the contribution!

I would like to add a test, give me some days to commit it.

**Comment by magnusburton** [comment]
Finally managed to fix the queries. I did however make a mistake and include commit 0c0b3fa. It does not belong in this PR

**Comment by derhuerst** [comment]
Thanks!

I have amended, squashed and split-up your commits for a cleaner Git history. Because I'm not allowed to push in your fork, I diverged from your fork, so this PR is displayed as closed instead of merged. Your commits are on `main` now though.

**Pull request closed without merging**